docs/protocol/index.md

```diff
@@ -4,55 +4,17 @@
+The on-chain **Autonolas Protocol** anchors the Autonolas autonomous services -and in particular the current incarnation as agent services- on the target settlement layer and provides the primitives needed to create, operate and secure such services.
+The protocol also provides a mechanism that incentivizes their creation and **rewards developers and service operators proportionally for their efforts to support the growth of the Autonolas ecosystem**.
 
-!!! info
-
-    The Autonolas Protocol is currently deployed in Ethereum and Goerly chains, and it will be progressively deployed in all major blockchains.
-
-From an architectural point of view, the smart contracts that make up the Autonolas Protocol satisfy the following properties:
-
-* Follow a core-periphery architecture (such as in Uniswap), which allows for changing out periphery functionality without changing the data models at the core.
-* Allow for extension via modules (such as in MakerDAO).
+<figure markdown>
+![Package minting](./images/ecosystem_flywheel.svg)
+</figure>
 
-Examples of modules include governance and staking. Governance is particularly important in a modular system, as it is used to vote on the adoption or abandoning of modules. By ensuring an immutable core, the Autonolas protocol provides guarantees that once created, the ecosystem’s agent components, canonical agents, and services are not mutable by governance – an important guarantee of censorship resistance.
+Autonolas strives to incentivize software composability and reusability. Thus, the Autonolas Protocol reflects such composability by representing software components as NFTs, and incorporates a mechanism that enables combining them to take into account their actual software composition relationship. This is a crucial feature to measure their contribution and reward according to the protocol tokenomics model.
 
-The Autonolas Protocol is built with the {{open_autonomy}} framework in mind as the primary framework for realizing agent services. However, it does not prescribe the usage of the {{open_autonomy}} framework and allows for services to be implemented on alternative frameworks.
+The Autonolas Protocol is [currently deployed](./technical_overview.md#contract-addresses) in Ethereum and Görli chains, and it will be progressively deployed in all major blockchains.
 
 !!! abstract "Learn more"
 
     Read the **Technical Architecture**, **Tokenomics** and **Governance** sections in the [Autonolas Whitepaper](https://www.autonolas.network/documents/whitepaper/Whitepaper%20v1.0.pdf) for the full details of the **Autonolas Protocol**.
-
```

Comment:
Do you think we can or have to cite the various repository and documentation therein?

Reply:
Sorry, I didn't make my point clear. 

I meant, as "owner" of this info textbox (which I also agree is useful here), do you find approppriate to insert the links to the details and technical repos (as suggested above by @mariapiamo) in this short text ?

Reply:
Ah I see, no that doesn't seem necessary to me